packages/rxfire/database/fromRef.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { database } from 'firebase';
+import { Observable } from 'rxjs';
+import { map, delay, share } from 'rxjs/operators';
+import { ListenEvent, SnapshotPrevKey } from './interfaces';
+
+/**
+ * Create an observable from a Database Reference or Database Query.
+ * @param ref Database Reference
+ * @param event Listen event type ('value', 'added', 'changed', 'removed', 'moved')
+ */
+export function fromRef(
```

Comment:
This method allows users to dynamically invoke various methods off of a reference. It's kind of scary and I would rather us support only the `on` and have users handle the `once` case themselves.

Reply:
fixed